admin_login_page.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="/css/common.css">
+    <link rel="stylesheet" type="text/css" href="/css/header.css">
+    <link rel="stylesheet" type="text/css" href="/css/kemi.css">
+    <link rel="stylesheet" type="text/css" href="/css/login_page.css">
+    <script src="https://kit.fontawesome.com/aa4811ab5f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Login - Page</title>
+</head>
+<body>
+    <div class="container">
+        <header id="menu_header" >
+            <!--<div class="logo">
+                <a href="index.html"><img src="images/logo3.PNG" alt="My logo"></a>
+                </div>  -->
+                <div class="kemi">
```

Comment:
try here to use good name for this class

Reply:
Have changed the class name to logo animation; also renamed the associated css file from kemi.css to logo_animation.css